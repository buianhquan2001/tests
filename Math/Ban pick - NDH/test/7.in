10
271 0.8411
291 0.0013
298 0.7342
265 0.0532
250 0.4177
275 0.1134
300 0.2353
255 0.172
273 0.3623
263 0.4317
