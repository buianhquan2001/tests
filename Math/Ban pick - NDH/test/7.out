177
1
162
10
72
21
48
30
68
78
